docs/contributing_to_qiskit.rst

```diff
@@ -7,693 +7,81 @@ Qiskit is an open-source project committed to bringing quantum computing to
-the PR isn't in its final state and will change. It also means that we will
-not merge the commit until it is finished. You or a reviewer can remove the
-[WIP] tag when the code is ready to be fully reviewed for merging.
+If you're not sure what type of contribution is right for you, take a look at the following flowchart to help you:
 
+.. raw:: html
+   :file: images/contributor-flowchart.svg
 
+.. _contributing_links:
 
-Code Review
-===========
+********************************
+Contributing to a Specific Repo
+********************************
 
-Code review is done in the open and is open to anyone. While only maintainers have
-access to merge commits, community feedback on pull requests is extremely valuable.
-It is also a good mechanism to learn about the code base. You can
-view a list of all open pull requests here:
+Each Qiskit package has it's own set of Contributing Guidelines (kept in the CONTRIBUTING.md file) which
+details specific information on contributing to that repository. Make sure you read through the repo-specific
+Contributing Guidelines prior to making your contribution to a specific repo as each project may have
+slightly different requirements and processes.
 
 =========================== =============================================
-Element                     Pull Requests
+Repo                        Contributing Guidelines
 =========================== =============================================
-qiskit-terra                https://github.com/Qiskit/qiskit-terra/pulls
-qiskit-aer                  https://github.com/Qiskit/qiskit-aer/pulls
-qiskit-ignis                https://github.com/Qiskit/qiskit-ignis/pulls
-Docs or Qiskit Meta-package https://github.com/Qiskit/qiskit/pulls
+Qiskit Terra                https://github.com/Qiskit/qiskit-terra/blob/main/CONTRIBUTING.md
+Qiskit Aer                  https://github.com/Qiskit/qiskit-aer/blob/main/CONTRIBUTING.md
+Qiskit Nature               https://github.com/Qiskit/qiskit-nature/blob/main/CONTRIBUTING.md
+Qiskit Optimization         https://github.com/Qiskit/qiskit-optimization/blob/main/CONTRIBUTING.md
+Qiskit Machine Learning     https://github.com/Qiskit/qiskit-machine-learning/blob/main/CONTRIBUTING.md
+Qiskit Finance              https://github.com/Qiskit/qiskit-finance/blob/main/CONTRIBUTING.md
+Qiskit Experiments          https://github.com/Qiskit/qiskit-experiments/blob/main/CONTRIBUTING.md
+Qiskit Dynamics             https://github.com/Qiskit/qiskit-dynamics/blob/main/CONTRIBUTING.md
+Qiskit Metal                https://github.com/Qiskit/qiskit-metal/blob/main/CONTRIBUTING.md
+
+Qiskit (meta-package)       https://github.com/Qiskit/qiskit/blob/master/CONTRIBUTING.md
+
+Qiskit Textbook (legacy)    https://github.com/qiskit-community/qiskit-textbook/blob/main/CONTRIBUTING.md
+Qiskit Textbook (beta)      https://github.com/Qiskit/platypus/blob/main/CONTRIBUTING.md
+
+Retworkx                    https://github.com/Qiskit/retworkx/blob/main/CONTRIBUTING.md
```

Comment:
I am not sure if Qiskit meta package should be after metal or in the first one. You don't want to put it the first because people will get confused it with Terra and draw too much traffic there?

Should retworkx be listed above textbook? Can we also add qiskit-tutorials repo, before textbook?

Reply:
for the metapackage i was following the order that was in the original contributing guidelines, and yes i don't want to draw too much traffic. 

I can add the tutorials repo but i'd rather add that one at the end as it is more for documentation/education contributions rather than code.

For retworkx I just put it at the end bc it is not strictly qiskit or quantum themed lol so wanted to make it separate, but it doesn't really matter.

I'll see if i can reorganise things a bit better 🤔

Reply:
I agree that tutorials repo is more for docuementation similar to textbooks. But I think it should be listed just above the textbook because it's "closer" / supposedly more up-to-date with latest Qiskit.

Reply:
I made the changes 😄 

also I think the tutorials repo contributing guidelines should be updated at some point soon if we're going to link to it here